src/openms/source/ANALYSIS/MAPMATCHING/QTClusterFinder.cpp

```diff
@@ -204,7 +204,7 @@ namespace OpenMS
          map_it != input_maps.end(); ++map_it)
     {
       max_intensity = max(max_intensity, map_it->getMaxInt());
-      max_mz = max(max_mz, map_it->getMax()[0]);
+      max_mz = max(max_mz, map_it->getMax()[1]);
```

Comment:
you can also use ` map_it->getMax().getY()` - this only works for D==2, so should be a natural safeguard in case someone decides to add an ion mobility dimension or whatever..